src/components/DraggableFlatList.tsx

```diff
@@ -244,6 +248,18 @@ function DraggableFlatListInner<T>(props: DraggableFlatListProps<T>) {
     [isTouchActiveNative, onDragEnd]
   );
 
+  useAnimatedReaction(
+    () => {
+      return spacerIndexAnim.value;
+    },
+    (cur, prev) => {
+      if (cur !== prev && cur >= 0 && prev >= 0) {
```

Comment:
let's remove the index check here and just keep the `if (cur !== prev)`. If consumers want to opt out of 0-index behavior they can do that within `onPlaceholderIndexChange`